jest.config.js

```diff
@@ -12,12 +12,12 @@ module.exports = {
     // @TODO: Look into helpers for this (e.g. https://github.com/mwolson/jest-webpack-alias)
     // Protons use a slightly different pattern because it's the only source pattern directory that
     // has a base-level index.js (this is how all components ensure access to base-level styles & variables).
-    '^protons[/]?(.*)': '<rootDir>/source/_patterns/00-protons/$1',
-    '^atoms/(.+)': '<rootDir>/source/_patterns/01-atoms/$1',
-    '^molecules/(.+)': '<rootDir>/source/_patterns/02-molecules/$1',
-    '^organisms/(.+)': '<rootDir>/source/_patterns/03-organisms/$1',
-    '^templates/(.+)': '<rootDir>/source/_patterns/04-templates/$1',
-    '^pages/(.+)': '<rootDir>/source/_patterns/05-pages/$1',
+    '^protons[/]?(.*)': '<rootDir>/source/default/_patterns/00-protons/$1',
+    '^atoms/(.+)': '<rootDir>/source/default/_patterns/01-atoms/$1',
+    '^molecules/(.+)': '<rootDir>/source/default/_patterns/02-molecules/$1',
+    '^organisms/(.+)': '<rootDir>/source/default/_patterns/03-organisms/$1',
+    '^templates/(.+)': '<rootDir>/source/default/_patterns/04-templates/$1',
+    '^pages/(.+)': '<rootDir>/source/default/_patterns/05-pages/$1',
```

Comment:
Certainly here you want another variable for "source/default"

Reply:
See above comments, give me some air here.